### PR TITLE
Add flag to control Output behaviour

### PR DESCRIPTION
**Pull request by chrisdunelm**
Updates #3433

### CODE REVIEW

File: tools/Google.Cloud.Tools.VersionCompat/Program.cs [hunk 1]
```diff
@@ -93,12 +93,26 @@ private enum ExitCodeBehavior
             MajorAndMinor = 2,
         }
 
+        private enum OutputBehavior
```
Comment (jskeet) [inline tools/Google.Cloud.Tools.VersionCompat/Program.cs]: Possibly make this a flags enum, then you can conditionalize it more easily?

Comment (chrisdunelm) [reply tools/Google.Cloud.Tools.VersionCompat/Program.cs]: done

File: tools/Google.Cloud.Tools.VersionCompat/Program.cs [hunk 2]
```diff
@@ -137,7 +154,7 @@ static int Main(string[] args)
 
             var diffs = Check(assem0, assem1, null);
 
-            if (diffs.Major.Any())
+            if ((options.OutputBehavior & OutputBehavior.Major) != 0 && diffs.Major.Any())
```
Comment (jskeet) [inline tools/Google.Cloud.Tools.VersionCompat/Program.cs]: Could use Enum.HasFlag, but that could come later.

Comment (chrisdunelm) [reply tools/Google.Cloud.Tools.VersionCompat/Program.cs]: ack :) I'd forgotten that existed!

**Pull request merged**